### DISCUSSION TITLE
https://demonkingswarn.is-a.dev/blog/2024/08/29/why-i-dont-care-about-ai

### DISCUSSION THREAD

**Discussion by giscus** [DemonKingSwarn/blog#11] [General]
# https://demonkingswarn.is-a.dev/blog/2024/08/29/why-i-dont-care-about-ai

In this blog I am going to talk about AI and how it's the new silicon valley snake oil.

https://demonkingswarn.is-a.dev/blog/2024/08/29/why-i-dont-care-about-ai

**Comment by RaghavGohil** [comment]
The word 'AI' makes me sick ngl. It's because people have hyped it so much that every new technology has 'AI' in it. I swear, if I see another ad on learning ChatGPT and other "free" AI tools, I'll break my laptop and never buy one again. 

Most of the tools aren't even free. They cost you money after you spend the credits. I pity those who buy AI courses and also never find out that it's a full-blown scam.

**Reply by hemanth2004** [reply]
True

**Reply by RaghavGohil** [reply]
Hi bro how have you been?
